doc/design/reader/README.md

```diff
@@ -1,35 +1,35 @@
-# Python Data Reader Design Doc
+# Python Data Reading Design Doc
 
-Paddle reads data from data reader during training. It will be passed into `paddle.train` as a parameter.
+Paddle reads data from *data reader* during training. *data reader creator* (or *reader creator*) creates a *data reader* when invoked. *reader creator* will be passed into `paddle.train` as a parameter.
```

Comment:
At training and testing time, PaddlePaddle programs need to read data. To ease the users' work to write data readign code, we define that

- A *reader* is a function that reads data (from file, network, random number generator, etc) and yields data items.
- A *reader creator* is a function that returns a reader function.
- A *reader decorator* is a function, which accepts one or more readers, and returns a reader.

and provide frequently used reader creators and reader decorators.